radixdlt-java/src/main/java/com/radixdlt/client/core/RadixUniverse.java

```diff
@@ -171,4 +172,8 @@ public void disconnect() {
 		ledger.close();
 		network.close();
 	}
+
+	public JsonObject toJson() {
```

Comment:
`RadixUniverse`, as defined in the lib as ledger+network, should not be serializable. Instead, expose method to retrieve config object.

Reply:
I updated this